src/server/scripts/Northrend/Naxxramas/boss_patchwerk.cpp

```diff
@@ -15,32 +15,38 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+/*
+ * Move everything from EVENT_HATEFUL to SPELL_HATEFUL_STRIKE_PRIMER spell script
+ * He should call trash(from the room with Embalming Slime up to frogger(not sure
+  if trash before Embalming Slime too or not)) after aggro
+ */
```

Comment:
same with the other PR, this doesn't feel like the place to leave this type of comment